Mathlib/Algebra/Module/LinearMap.lean

```diff
@@ -1246,3 +1246,9 @@ theorem End.intCast_def (z : ℤ) [AddCommGroup N₁] [Module R N₁] :
 #align module.End.int_cast_def Module.End.intCast_def
 
 end Module
+
+/-- `Function.eval` as a linear map. -/
+@[simps!] def Pi.evalₗ (I : Type u)  (f : I → Type v) (R) [Semiring R]
```

Comment:
This is [LinearMap.proj](https://leanprover-community.github.io/mathlib4_docs/Mathlib/LinearAlgebra/Pi.html#LinearMap.proj). Yes, the naming is a bit inconsistent...

Reply:
That's annoying! Should I add an alias so we have both names?

Reply:
Adding an alias would only postpone the problem, since all lemmas about it will still have `proj` in their name. Granted, that's a bit better for discoverability, but I think the real solution is just to change this name if people on Zulip agree (and in an other PR of course!)

Reply:
Ok, I've created a [poll](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/LinearMap.2Eproj/near/385549555) on Zulip. Now that I think about it, the `proj` name seems more natural for dependent functions, but `eval` for non-dependent ones.